bin/datasource-invoke.js

```diff
@@ -22,9 +22,15 @@ process.once('message', function(msg) {
     send({
       error: err,
       callbackArgs: args,
-    });
-
-    process.nextTick(function() {
+    }, function() {
+      // XXX better to do this when the send has completed, which may be the
+      // actual source of your bug.
+      //
+      // But even better would be to not do this here. kill your child in your
+      // parent process after its received its message (assuming that you don't
+      // control the invoked methods... they shouldn't have side effects, and
+      // node should exit by itself, but if you don't trust them, best to kill
+      // from the parent where it cannot be stopped).
```

Comment:
Many LoopBack connectors create a connection pool at setup time, I think that's why we added `process.exit()` call here.

> kill your child in your parent process after its received its message

Sounds like the right approach to me.